probe-rs/Cargo.toml

```diff
@@ -46,4 +46,9 @@ includedir = "0.5.0"
 quote = "1.0.2"
 log = "0.4.6"
 serde_yaml = "0.8.11"
-proc-macro2 = "1.0.4"
\ No newline at end of file
+proc-macro2 = "1.0.4"
+
+[dev-dependencies]
+rand = "0.7.2"
+structopt = "*"
```

Comment:
We should use a fixed version here to a) avoid breakage and b) to avoid the cargo bug where it would rebuild the crate even if you didn't change anything.